.github/workflows/dotnetcore.yml

```diff
@@ -0,0 +1,35 @@
+name: .NET Core
```

Comment:
The inclusion of a workflows config is a bit surprising. What is the reasoning for this?

Reply:
I have add it to find out why some of SevenZipCompressorTests.cs tests failed at AppVeyor
but then I fix tests at a5d0dd4. If you do not plan to use github workflow for tests can delete it